docs/source/getting-started/installation.rst

```diff
@@ -7,61 +7,122 @@ Installation
-  $ # pip install "skypilot[all]"
+        .. code-block:: shell
 
-SkyPilot currently supports these cloud providers: AWS, GCP, Azure, IBM, OCI, SCP, Lambda Cloud, and Cloudflare (for R2 object store).
-If you only have access to certain clouds, use any combination of
-the pip extras above (e.g., :code:`"[aws,gcp]"`) to reduce the
-dependencies installed.
+          # SkyPilot requires python >= 3.7. For Apple Silicon, use >= 3.8.
+          # Recommended: use a new conda env to avoid package conflicts.
+          conda create -y -n sky python=3.8
+          conda activate sky
 
+          pip install skypilot-nightly
 
-To get the latest features/updates, either install the nightly build:
+          # Or, choose an extra (defaults to [aws]):
+          pip install "skypilot-nightly[aws]"
+          pip install "skypilot-nightly[gcp]"
+          pip install "skypilot-nightly[azure]"
+          pip install "skypilot-nightly[kubernetes]"
+          pip install "skypilot-nightly[lambda]"
+          pip install "skypilot-nightly[ibm]"
+          pip install "skypilot-nightly[scp]"
+          pip install "skypilot-nightly[oci]"
+          pip install "skypilot-nightly[all]"
 
-.. code-block:: console
+    .. tab-item:: Latest Release
+
+        .. code-block:: shell
+
+          # SkyPilot requires python >= 3.7. For Apple Silicon, use >= 3.8.
+          # Recommended: use a new conda env to avoid package conflicts.
+          conda create -y -n sky python=3.8
+          conda activate sky
+
+          pip install skypilot
+
+          # Or, choose an extra (defaults to [aws]):
+          pip install "skypilot[aws]"
+          pip install "skypilot[gcp]"
+          pip install "skypilot[azure]"
+          pip install "skypilot[kubernetes]"
+          pip install "skypilot[lambda]"
+          pip install "skypilot[ibm]"
+          pip install "skypilot[scp]"
+          pip install "skypilot[oci]"
+          pip install "skypilot[all]"
+
+To use a subset of the clouds, you can combine some of the pip extras above to
+reduce the dependencies installed:
 
-  $ pip install -U "skypilot-nightly[all]"
+.. code-block:: shell
 
-or install from source:
+  # Nightly:
+  pip install -U "skypilot-nightly[aws,gcp]"
+  # Latest release:
+  pip install -U "skypilot[aws,gcp]"
+
+To get the latest features, either use a nightly build (the **Nightly**
+tab above), or install from source:
 
 .. code-block:: console
 
   $ git clone https://github.com/skypilot-org/skypilot.git
   $ cd skypilot
   $ pip install ".[all]"
 
+As an alternative to installing SkyPilot on your laptop, we also provide a Docker image as a quick way to try out SkyPilot. See instructions below on running SkyPilot :ref:`in a container <docker-image>`.
 
+.. _verify-cloud-access:
 
-.. note::
+Verifying cloud access
+------------------------------------
+
+After installation, run :code:`sky check` to verify that credentials are correctly set up:
+
+.. code-block:: console
+
+  $ sky check
```

Comment:
We are having `$` for all the commands below, but not for the command for installation above. Is that expected?

Reply:
Changed console -> shell and removed `$` everywhere on this page. Feels cleaner.

Reply:
Good point. The install page needs to reduce friction/text as much as possible, so opting to not keep `$`. For other pages, there's already some inconsistency which I think is okay (if some snippets have both commands and outputs, then it makes sense to `$`).